source/backend/dal/Repositories/AcquisitionFilePropertyRepository.cs

```diff
@@ -62,6 +63,11 @@ public PimsPropertyAcquisitionFile Add(PimsPropertyAcquisitionFile propertyAcqui
         {
             propertyAcquisitionFile.ThrowIfNull(nameof(propertyAcquisitionFile));
 
+            if (propertyAcquisitionFile.Property.IsRetired.HasValue && propertyAcquisitionFile.Property.IsRetired.Value)
```

Comment:
hmm, this would appear to not cover acquisition updates, just add. Was that intentional?